src/index.js

```diff
@@ -15,7 +16,6 @@ import {
   SessionSecretToken,
 } from './tokens';
 
-declare var __BROWSER__: Boolean;
 export default (__BROWSER__ ? browser : server);
 
 export {SessionCookieExpiresToken, SessionCookieNameToken, SessionSecretToken};
```

Comment:
While maximizing coverage for our plugins is a nice to have, the most impactful changes are those that support all of these exports being fully typed.  So while we don't have 100% coverage for this plugin, we **do** have full coverage of the exports for any consumers of this plugin.